ui/selectors/confirm-transaction.js

```diff
@@ -244,36 +283,52 @@ export const transactionFeeSelector = function (state, txData) {
     numberOfDecimals: 6,
   });
 
-  const hexTransactionFee = getHexGasTotal({ gasLimit, gasPrice });
+  const hexMinimumTransactionFee = getMinimumGasTotalInHexWei(
+    gasEstimationObject,
+  );
+  const hexMaximumTransactionFee = getMaximumGasTotalInHexWei(
+    gasEstimationObject,
+  );
 
-  const fiatTransactionFee = getTransactionFee({
-    value: hexTransactionFee,
+  const fiatMinimumTransactionFee = getTransactionFee({
```

Comment:
the old value, fiatTransactionFee appears to be unused everywhere. Probably for the best to rely upon our components for that. Will remove in a later PR and I wouldn't tap into these.